src/main/java/com/klarna/hiverunner/StandaloneHiveRunner.java

```diff
@@ -159,6 +159,9 @@ protected final void runTestMethod(FrameworkMethod method,
      */
     public HiveShellContainer evaluateStatement(List<? extends Script> scripts, Object target,
         Path temporaryFolder, Statement base) throws Throwable {
+        if(scripts==null){
```

Comment:
This is fixing a symptom, not the cause, I think if you set https://github.com/klarna/HiveRunner/blob/main/src/main/java/com/klarna/hiverunner/HiveRunnerRule.java#L38 to an empty list there it will make the rule more consistent with the extension and remove the need for this null check.